### ISSUE TITLE
Make Error: ChangeLog not found

### ISSUE THREAD

**Issue by j5shi** [tinyproxy/tinyproxy#37]
``` bash
$ ./autogen.sh
+ cd .
+ aclocal -I m4macros
+ autoheader
+ automake --gnu --add-missing
Makefile.am: error: required file './ChangeLog' not found
+ cd /root/usr/j5shi/tinyproxy
+ set -
./configure
./autogen.sh: line 21: ./configure: No such file or directory

Configure failed or did not finish!
```

Could you tell me where to get the `ChangeLog`? I cloned your repo and ran `./autogen.sh`.

**Comment by rscullin** [comment]
You can `touch ChangeLog` and then `./autogen.sh` will run fine.

**Comment by j5shi** [comment]
Thanks @rscullin, that works for me.

**Comment by rofl0r** [comment]
i think this should be kept open until the issue is properly fixed here in the repo.

**Comment by j5shi** [comment]
yep, so reopened. As I can't find the instruction from the guide, so either update the guide or fix it in  makefile.

**Comment by obnoxxx** [comment]
closed by PR #61 , duplicate of issue #60
